src/main/java/com/demo/classroom/Entity/Book.java

```diff
@@ -0,0 +1,29 @@
+package com.demo.classroom.Entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long bookId;
+
+    private String name;
+    private String author;
+
+    @ManyToOne
```

Comment:
Use lazy loading to improve performance.
```suggestion
    @ManyToOne(fetch = FetchType.LAZY)
```

Reply:
Done.